DesignPattern-RequestResponse/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+    "title": "Request Response over HTTP",
+    "description": "Receive a message over HTTP, perform some other work of your choosing, and then return a response",
+    "tags": [
+        "HTTP"
+
+    ],
+    "skus": [
+        "consumption",
+        "standard"
+    ],
+    "kinds": [
+        "stateful",
+        "stateless"
+    ],
+    "details_description": "This template serves the starting point for building synchronous Request Response patterns using Logic Apps. Learn more about these capabilites here -  https://learn.microsoft.com/azure/connectors/connectors-native-reqres?tabs=consumption ",
+    "details": {
+        "By": "Microsoft",
+        "Type": "Workflow",
+        "Trigger": "Request",
+        "Category": "Design Patterns , HTTP"
+    },
+    "artifacts": [
+        {
+            "type": "workflow",
+            "file": "workflow.json"
+        }
+    ],
+    "images": {
+        "workflow": "workflow.png"
```

Comment:
For images, we need "dark" and "light". Also, since we are hard-coding the image extension to .png now, we no longer want to include .png.

If you would have the light and dark images, that'd be better. For now, below is the syntax:

`"images": {
        "light": "workflow",
        "dark": "workflow"
    },`